packages/platform/examples/base/README.md

```diff
@@ -1,35 +1,16 @@
 # Atoms examples
 Example apps using atoms - customizable UI components to integrate scheduling into your product.
 
-## Running examples apps locally
-
-1. Cal's backend is required to run example apps, so clone `https://github.com/calcom/cal.com` and follow setup instructions in the readme. Importantly, you need to have Google credentials setup by following [this section](https://github.com/calcom/cal.com?tab=readme-ov-file#obtaining-the-google-api-credentials) in the docs.
+## Set up your sandbox
 
-2. Open "apps/api/v2/.env" and copy environment variables below, and then copy `NEXTAUTH_SECRET` from the root ".env" of repository cloned in step 1.
-```jsx
-NODE_ENV="development"
-API_PORT=5555
-DATABASE_READ_URL="postgresql://postgres:@localhost:5450/calendso"
-DATABASE_WRITE_URL="postgresql://postgres:@localhost:5450/calendso"
-API_URL="http://localhost"
-NEXTAUTH_SECRET="copy from .env of root of repository cloned in step 1"
-DATABASE_URL="postgresql://postgres:@localhost:5450/calendso"
-JWT_SECRET="asjdijI1JIO12I3O89198jojioSAJDU"
-REDIS_URL="redis://localhost:6379"
-```
+1. Sign up for a sandbox at "https://app.cal.dev/signup". This cannot be used for production.
+2. In the web app navigate to `https://app.cal.dev/settings/platform/new` and create a platform.
+3. In the web app navigate to `https://app.cal.dev/settings/platform/oauth-clients` and create a new oAuth client - give all permissions and set redirect uri to `http://localhost:4321` which points to example app.
+4. Start the example app by running `yarn dev` and go to `http://localhost:4321`.
+5. You can now use and customise the Cal.com Atoms
 
-3. Start "apps/api/v2" api using `yarn dev`.
-4. Start "packages/platform/atoms" atoms package with `yarn dev`.
-5. Start "apps/web" cal web app using `yarn dx`.
-6. Open your browser at "http://localhost:3000/" and login with admin username `admin@example.com` and password `ADMINadmin2022!`.
-7. In the web app navigate to `http://localhost:3000/settings/organizations/new` and create a sample organization. When asked for phone verification code enter `111111`.
```

Comment:
the goal of the example app is to test quickly. we should not ask the users to run the entire repo locally, thats the whole point of hosted platform, to not do that and only work with Atoms